src/software/mysql/MySQL.MD

```diff
@@ -0,0 +1,435 @@
+#MySQL  
+
+##简介  
+
+MySQL是一个关系型数据库管理系统
+
+由瑞典MySQL AB公司开发，属于Oracle旗下产品
+
+是最流行的关系型数据库管理系统之一，是最好的RDBMS关系数据库管理系统应用软件之一
+
+使用SQL语言，分为社区版和商业版，由于其体积小、速度快、总体拥有成本低，尤其是开放源码这一特点，一般中小型网站都选择MySQL作为网站数据库
+
+##安装  
+
+下载地址：https://dev.mysql.com/downloads/
+
+在这个下载界面会有几个版本的选择。
+
+1. MySQL Community Server 社区版本，开源免费，但不提供官方技术支持。
+2. MySQL Enterprise Edition 企业版本，需付费，可以试用30天。
+3. MySQL Cluster 集群版，开源免费。可将几个MySQL Server封装成一个Server。
+4. MySQL Cluster CGE 高级集群版，需付费。
+5. MySQL Workbench（GUITOOL）一款专为MySQL设计的ER/数据库建模工具。它是著名的数据库设计工具DBDesigner4的继任者。MySQLWorkbench又分为两个版本，分别是社区版（MySQL Workbench OSS）、商用版（MySQL WorkbenchSE）。
+
+MySQL Community Server是开源免费的，这也是我们通常用的MySQL的版本。
+
+1. 下载后得到zip压缩包.
+
+2. 解压到自己想要安装到的目录
+
+3. 添加环境变量：我的电脑->属性->高级->环境变量
+    选择PATH,在其后面添加: mysql 安装文件下面的bin文件夹
+
+4. 新建mysql配置文件
+
+    在mysql文件夹中新建my.ini文件,编辑 my.ini 文件,注意替换路径位置
+    ```
+    [mysqld]
+    basedir=D:\Program Files\mysql-5.7\     
+    datadir=D:\Program Files\mysql-5.7\data\
+    port=3306
+    skip-grant-tables
+    ```
+5. 启动管理员模式下的CMD，输入`cd /d bin目录路径`将路径切换至mysql下的bin目录，然后输入`mysqld –install` (安装mysql)

```

Comment:
We prefer to space the inline code.

Reply:
What is inline code qaq

Reply:
https://vuepress-theme.mrhope.site/basic/markdown/demo/#codes